go/bytes2/buffer_test.go

```diff
@@ -16,23 +16,45 @@ limitations under the License.
 
 package bytes2
 
-import (
-	"testing"
-)
+import "testing"
+
+// Checks if the actual value is equal to the expected value for the given function
+func assertEqual(t *testing.T, got any, want any, funcName string) {
+	if got != want {
+		t.Errorf("%s: got %v, want %v", funcName, got, want)
+	}
+}
```

Comment:
I think we can use `testify` here instead.

Reply:
Got it! Thanks for the review.